package.xml

```diff
@@ -1,11 +1,16 @@
 <package>
   <name>srdfdom</name>
-  <version>0.2.7</version>
+  <version>0.2.8</version>
   <description>Parser for Semantic Robot Description Format (SRDF).</description>
   <author email="isucan@willowgarage.edu">Ioan Sucan</author>
+  <author email="gwalck@techfak.uni-bielefeld.de">Guillaume Walck</author>
+  <author>Wim Meeussen</author>
+  <author>John Hsu</author>
```

Comment:
I think what you have in the README is sufficient, if we made everyone authors of new packages based on references to old code, we would have very long author lists. Generally, only major contributors to a package who have specifically worked on that one claim authorship. I think you should remove the other ones.